### DISCUSSION TITLE
2023/02/01/hetzner-storagebox-backups/

### DISCUSSION THREAD

**Discussion by giscus** [jdeluyck/kcoreorg-website#161] [Blog Comments]
# 2023/02/01/hetzner-storagebox-backups/

For my offsite backup strategy I’m relying on rclone and restic. Rclone is being used to encrypt/copy backups taken by Proxmox Backup Server (which takes backups of the VM’s/Linux Containers running on Proxmox VE, deduplicates and compresses the data) to remote storage, while restic is used to do deduplication/encryption and copying of data stored on my NAS to remote storage.

https://kcore.org/2023/02/01/hetzner-storagebox-backups/

**Comment by JackBrenn** [comment]
Hi,

Just to clarify. Are you sending the contents of your Proxmox Backup Server (PBS) datastore (/mnt/datastore/<datastorename>) to the Hetzner Storage box?

**Reply by jdeluyck** [reply]
For PBS I'm indeed doing that. More precisely, I create a snapshot, clone it (so that I always have the same path) and then rclone that data off to the remote storage.

PBS already uses dedup and compression so I'm not going to add that on top - I do add encryption.
